testutil-server/src/test/java/io/spine/server/aggregate/given/AggregateCommandTestShouldEnv.java

```diff
@@ -0,0 +1,200 @@
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.aggregate.given;
+
+import com.google.protobuf.Message;
+import com.google.protobuf.Timestamp;
+import io.spine.server.aggregate.Aggregate;
+import io.spine.server.aggregate.AggregateCommandTest;
+import io.spine.server.aggregate.AggregateRepository;
+import io.spine.server.aggregate.Apply;
+import io.spine.server.command.Assign;
+import io.spine.server.entity.Repository;
+import io.spine.server.entity.given.Given;
+import io.spine.server.expected.CommandHandlerExpected;
+import io.spine.test.testutil.TUAssignProject;
+import io.spine.test.testutil.TUCreateProject;
+import io.spine.test.testutil.TUFailedToCreateProject;
+import io.spine.test.testutil.TUProjectAggregate;
+import io.spine.test.testutil.TUProjectAggregateVBuilder;
+import io.spine.test.testutil.TUProjectCreated;
+import io.spine.test.testutil.TUProjectId;
+import org.junit.jupiter.api.BeforeEach;
+
+import static com.google.protobuf.util.Timestamps.fromMillis;
+
+/**
+ * @author Vladyslav Lubenskyi
+ */
+public class AggregateCommandTestShouldEnv {
+
+    private static final TUProjectId ID = TUProjectId.newBuilder()
+                                                     .setValue("project id util")
+                                                     .build();
+
+    /**
+     * Prevents instantiation of this utility class.
+     */
+    private AggregateCommandTestShouldEnv() {
+    }
+
+    public static CommandHandlingAggregate aggregate() {
+        CommandHandlingAggregate result = Given.aggregateOfClass(CommandHandlingAggregate.class)
+                                               .withId(ID)
+                                               .withVersion(64)
+                                               .build();
+        return result;
+    }
+
+    /**
+     * A dummy aggregate that handles two command messages:
+     *
+     * <ul>
+     *     <li>accepts {@code TUCreateProject}.
+     *     <li>rejects {@code TUAssignProject}.
+     * </ul>
+     */
+    public static final class CommandHandlingAggregate
+            extends Aggregate<TUProjectId,
+                              TUProjectAggregate,
+                              TUProjectAggregateVBuilder> {
+
+        CommandHandlingAggregate(TUProjectId id) {
+            super(id);
+        }
+
+        @Assign
+        public TUProjectCreated handle(TUCreateProject command) {
+            return TUProjectCreated.getDefaultInstance();
+        }
+
+        @Assign
+        public Timestamp handle(TUAssignProject command)
```

Comment:
Please fix this method so that it makes sense in terms of everything. And then LGTM.